Detectors/MUON/MCH/Tracking/src/TrackFinderOriginal.cxx

```diff
@@ -67,10 +67,17 @@ const std::list<Track>& TrackFinderOriginal::findTracks(const std::array<std::li
 
   // Look for candidates from clusters in stations(1..) 4 and 5
   print("\n--> Step 1: find track candidates\n");
+  auto tStart = std::chrono::high_resolution_clock::now();
```

Comment:
Is not such frequent time query in a tight loop degrade the performance? I think in such cases it is better to have a special "time profiling" mode, activated either by e.g. ``if (timeProfing) {...`` or even at compilation level ``#ifdef TIME_PROFILING ... #endif``

Reply:
Yes but they are not in a tight loop, they are called only once per event. I did not notice a change in the overall timing when adding these timers and I don't see them when profiling the code with instruments.

Reply:
Indeed, I had 1st impression they are called once per track, sorry for the noise...